_posts/2024/2024-02-06-about-caching-in-eo.md

```diff
@@ -0,0 +1,237 @@
+---
+layout: post
+date: 2024-02-06
+title: "Build cache in EO and other build systems"
+author: Alekseeva Yana
+---
+
+
+## Introduction 
+In [EO](https://github.com/objectionary/eo) a caching is used to speed up program execution.
+While developing [EO](https://github.com/objectionary/eo) we found a caching 
```

Comment:
@Yanich96 No need to mention `EO` twice:
```
While developing [EO](https://github.com/objectionary/eo)
```
In the previous sentence you already mentioned it. Just:
```
Recently we found a error...
```
, btw a "error"? Maybe it's just a "bug"?